stylus/utilities/text.styl

```diff
@@ -174,11 +174,29 @@ fontstyle = {
     'fs-normal': 'fs-normal',
     'fs-italic': 'fs-italic'
 }
-
 if cssmodules == true
     cssModulesUtils(fontstyle, breakpoints)
 else
     nativeUtils(fontstyle, breakpoints)
+
+fw-normal()
+    font-weight normal
+
+fw-bold()
+    font-weight bold
```

Comment:
We have a `Bold` Component (https://github.com/cozy/cozy-ui/blob/master/react/Text/index.jsx#L59). Shouldn't we be coherent with it? ping @y-lohse

Reply:
Why do you think it's incoherent? They both use a font-weight of `bold` 🤔

Reply:
Because `<Bold>` uses `title-h4`. This class : 
- Sets font-size 
- deals with small-screen 
- sets text-color 

So I'm wondering if `u-bold` should not do the same thing?

Reply:
Ah, I see. I think it's a mistake for bold to use `title-h4` in the first place though. It shouldn't be coupled to a title class, but more importantly, utility classes are not components. It's ok for a component to set multiple css rules, but the idea of utility-classes is that they only do one thing and that you combine them as you need. So I think it's right that `Bold` component and `u-fw-bold` utility classes don't do the same thing.

Reply:
Thanks!